dynamic-mapping-ui/src/extension/grid/extension.component.ts

```diff
@@ -42,22 +43,30 @@ export class ExtensionComponent implements OnInit, OnDestroy {
   extensions$: Observable<IManagedObject[]>;
 
   listClass: string;
+  feature: Feature;
 
   constructor(
     private bsModalService: BsModalService,
     private extensionService: ExtensionService,
     private brokerConfigurationService: BrokerConfigurationService,
-    private alertService: AlertService
+    private alertService: AlertService,
+    private sharedService: SharedService
   ) {}
 
   async ngOnInit() {
-    this.extensions$ = this.reload$.pipe(
-        tap(() => (this.reloading = true)),
-        switchMap(() => this.extensionService.getExtensionsEnriched(undefined)),
-        tap(console.log),
-        tap(() => (this.reloading = false)),
-        shareReplay()
+    this.feature = await this.sharedService.getFeatures();
+    if (!this.feature.userHasMappingAdminRole) {
+      this.alertService.warning(
+        "The configuration on this tab is not editable, as you don't have Mapping ADMIN permissions. Please assign Mapping ADMIN permissions to your user."
```

Comment:
To make it more readable I would change "Mapping ADMIN" to "MAPPING ADMIN" (so all role related info as capital letters)